app/javascript/mastodon/reducers/notification_groups.ts

```diff
@@ -51,6 +52,7 @@ interface NotificationGroupsState {
   readMarkerId: string;
   mounted: number;
   isTabVisible: boolean;
+  mergedNotifications: 'ok' | 'pending' | 'needs-reload';
```

Comment:
It never gets set to `pending`? Maybe you could add a comment explaining what those values mean, and what it is used for

Reply:
It currently doesn't get set to pending, indeed. My intention was that accepting a notification request would mark it as pending, so that the polling code (which does not exist yet) knows there is a reason to poll.

So basically the flow would be:
- accepting a request switches to `pending`
- receiving `notifications_merged` or polling the endpoint changes to `needs-reload`
- loading the notifications changes to `ok`